Add multi-delete docs (#278)
freedomofpress/securedrop-workstation-docs: docs/journalist/sources.rst

  with a specific source (messages, files, and journalist replies) will be
  displayed here.

> - The top panel holds a toolbar that allows you to apply actions in bulk.

Reviewer: Wondering if "bulk" is a meaningful term here. I feel like if I were to encounter this organically, it might not make sense to me. It's also the only time the term is used here in the docs.

Maybe?:

```suggestion
- The top panel holds a toolbar that allows you to apply actions to multiple items at once.
```

|screenshot_send_reply|

Deleting conversations
---------------------- 

Deleting a single conversation
''''''''''''''''''''''''''''''

You can delete a single source conversation by clicking on the three dots at

> the top righthand side of the application window, beside the timestamp. You should

Reviewer: ```suggestion
the top right-hand side of the application window, beside the timestamp. You should
```
